lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -40,12 +40,18 @@ class Map
   end
 end
 
-# Patch the gcloud command used by googleauth to avoid spamming stderr.
 module Google
   module Auth
+    # Patch the gcloud command used by googleauth to avoid spamming stderr.
     module CredentialsLoader
+      # Set $VERBOSE to nil to mute the following warning:
+      # "warning: already initialized constant Google::Auth::CredentialsLoader
+      # ::GCLOUD_CONFIG_COMMAND"
```

Comment:
Should we keep the warning text on one line? Or at least keep the full constant name on one line?

Reply:
Changed